src/middlewares/roomsAvailability.js

```diff
@@ -24,8 +49,8 @@ const checkForRooms = async (req, res, next) => {
   });
 
   if (unavailableRooms.count > 0) {
-    const bookedRooms = unavailableRooms.rows.map(room => room.id);
-    return Responses.handleSuccess('409', 'rooms already booked', res, { unAvailableRooms: bookedRooms });
+    const bookedRoom = unavailableRooms.rows.map(room => room.id);
```

Comment:
I think the `bookedRoom` should be in plural `bookedRooms` as it's an array of many ids.